README.md

```diff
@@ -1 +1,49 @@
-# Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Portfolio is one of the most powerful tools in software developer toolbox. it is an easy way of showing what you are truly capable of as a developer, and a quick way for for recruiters to and hiring managers to get an idea of what you can bring to their team
+
+
+## Built With
+
+- HTML
+- CSS
+- Node.js
+- Webhint, Stylelint
+
+## Getting Started
+
+### RUN Project
+#### - Clone project 
+```
+git clone https://git@github.com:Jeanbulambo/Portfolio-repo.git
+```
+#### - Open project directory and open index.html file with your prefered browser
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@Jeanbulambo](https://github.com/Jeanbulambo)
+
```

Comment:
- [ ] [OPTIONAL] I think it would be better to add some more sources to increase your network, You may add twitter and linkedin